assets/data/awards.json

```diff
@@ -0,0 +1,26 @@
+[
+    {
+        "name": "Awards",
+        "titles": [
+            "Most Outstanding Social/Support Organization",
+            "Student Alumni Engagement Sponsorship"
+        ],
+        "providers": [
+            "Anteater Awards 2011",
+            "Student Alumni Association"
+        ]
+    },
+
+    {
+        "name": "Recognitions",
+        "titles": [
+            "Third Place",
+            "Second Place"
+        ],
+        "providers": [
+            "Winter UCI Alumni Association Club Affiliates Challenge 2020",
+            "UCI Campus Organizations & Volunteer Programs | Love My Org Contest 2020"
+        ]
+
+    }
+]
```

Comment:
I think you're right that we can have this in one file.

However I would restructure it a bit so it's more clear. Something like:

```json
{
  "awards": [
    {
      "title": "...",
      "desc": "...",
      "provider": "..."
    }
  ],

  "recognitions": [
    { ... },
    ...
  ]
}

```

This way awards and recognitions are clearly separated and each award is it's own object (instead of having to worry about indexing arrays correctly)

Reply:
Just finished restructuring the JSON file to the example you gave!